src/_incydr_sdk/sessions/client.py

```diff
@@ -76,9 +76,9 @@ def get_page(
 
         # Parse timestamps
         if start_time and not isinstance(start_time, (int, float)):
-            start_time = parse_ts_to_posix_ts(start_time)
+            start_time = parse_ts_to_posix_ts(start_time) * 1000
```

Comment:
Since we have to do this in a few places, it might be nice to have another parse method that returns the timestamp in ms.

Reply:
^ Was going to say something similar - the approach here is totally fine, but maybe having a method that explicitly names what we're doing might be helpful if we come back in the future and are like "wait why are we multiplying by 1000"

Reply:
That's a good point. I'll add something.